web/impact/impact/views/algolia_api_key_view.py

```diff
@@ -69,33 +70,38 @@ def _get_search_key(request):
 
 def _get_filters(request):
 
+    if request.GET['index'] == 'people':
```

Comment:
could this be tested?

Reply:
i'd love to see this as well but i'm skeptical we have an infrastructure that allows us to test 3rd party stuff well. this might be a good candidate for unit test given the circumstances

Reply:
sorry I didn't catch this sooner 😩 I tested this logic